CONTRIBUTING.md

```diff
@@ -71,12 +71,32 @@ Filters take the document as parameter and are:
 
 Filters are loaded automatically from files named after the service they operate on. For example, filters for the Meetup service, which is declared in `services/Meetup.json`, are loaded from `services/Meetup.filters.js`.
 
-Each filter is exposed as a named function export that takes a `document` parameter and behaves like the `document` object in a browser DOM.
+Each filter is exposed as a named function export, which could be `async`, that takes a `document` parameter and behaves like the `document` object in a browser DOM.
 
 > It is actually a [JSDOM](https://github.com/jsdom/jsdom) document instance.
 
+There is also an optional parameter containing the whole document declaration. It can be useful if you need to access the defined document url or selector inside your filter.
+
+For example, you can scope your the selection in your filter by the `selector` defined in the document declaration:
+
+```js
+export function removeImages(document, { select: selector }) {
+  const images = document.querySelectorAll(`${selector} img`);
+  images.forEach(el => el.remove());
```

Comment:
> This documentation is very important as it sets the standard for what we expect 🙂
> Do we want to encourage this or `document.querySelectorAll(`${selector} img`).forEach(el => el.remove());`?
> 
> It's a real question, I'm not sure I have an opinion on this.

Reply:
I do not have a strong opinion on this neither, so I suppose it is no so important :).